scripts/rmc.py

```diff
@@ -102,7 +105,26 @@ def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False):
     cbmc_cmd = ["cbmc"] + cbmc_args + [cbmc_filename]
     return run_cmd(cbmc_cmd, label="cbmc", output_to="stdout", verbose=verbose, quiet=quiet)
 
+def cbmc_viewer_version_correct():
+    if not is_exe(CBMC_VIEWER):
+        print("ERROR: Could not find cbmc-viewer in PATH")
+        return False
+
+    version_cmd = [CBMC_VIEWER, "--version"]
+    version_output = subprocess.run(
+        version_cmd, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT).stdout
+        
+    parts = re.search("CBMC viewer (\d+\.\d+)", version_output.strip())
+    if float(parts.group(1)) < CBMC_VIEWER_REQUIRED_VERSION:
+        print(f"ERROR: Expected cbmc-viewer version {CBMC_VIEWER_REQUIRED_VERSION}")
+        return False
```

Comment:
Please define major and minor versions (as integers) and compare them as it is done in [this script](https://github.com/model-checking/rmc/blob/main-152-2021-06-07/scripts/check-cbmc-version.py).